package.json

```diff
@@ -26,6 +27,7 @@
     "react-native": "0.67.2",
     "react-native-cli": "^2.0.1",
     "react-native-picker-select": "^8.0.4",
+    "react-native-popup-dialog": "^0.18.3",
```

Comment:
do we need this?

Reply:
yes i am using it to show the popUp, import Dialog, {DialogTitle, DialogContent} from 'react-native-popup-dialog';

Reply:
but you didn't use it in this branch, it was from a previous branch?